subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -16,34 +16,72 @@
 [[what_is_gradle]]
 = What is Gradle?
 
-Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool flexible enough to build almost any type of software.
-Gradle makes few assumptions about what you're trying to build or how to build it.
-This makes Gradle particularly flexible.
+Gradle, Inc. is the software company behind Gradle Build Tool (www.gradle.org) and Gradle Enterprise (www.gradle.com).
```

Comment:
Hmmm I would be ok with just removing this line and start with the Gradle Build Tool section. Not sure how others feel.

Reply:
I made this change based on a suggestion by [ljacomet](https://github.com/ljacomet). I do need something that ties GE and BT together somewhere so users understand this is the BT manual.